bin/run_dwelling.py

```diff
@@ -34,7 +34,7 @@
     'verbosity': 6,                         # verbosity of time series files (0-9)
     # 'metrics_verbosity': 6,               # verbosity of metrics file (0-9), default=6
     # 'save_results': False,                # saves results to files. Defaults to True if verbosity > 0
-    'output_path': os.getcwd(),           # defaults to hpxml_file path
```

Comment:
@mnblonsky : We want this change in run_dwelling going forward? Seems like a reasonable one to me, just want to make sure we're being careful about what gets changed here. I actually think most users would expect .csv output rather than parquet by default.

Reply:
Yes, I think this is better for default behavior. I can comment out the to_parquet option too, I think that's a good plan.

Reply:
Done!